src/fides/api/ops/service/messaging/messaging_crud_service.py

```diff
@@ -29,7 +29,7 @@ def create_or_update_messaging_config(
     data = {
         "key": config.key,
         "name": config.name,
-        "service_type": config.service_type,
+        "service_type": config.service_type.value,
```

Comment:
`sqlalchemy` does some [auto-translation of enums in the background](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.Enum), so either should work just fine. It's just that if `.value` is passed in it will validate it as a string (because that's what `.value` is).

Reply:
> hm, was this broken before?

a good callout. i don't think it was.

my best guess is that i'd made this change _before_ i switched the variable names in the enum to lowercased (but after updating the enum values in the DB), and i was trying to make things work in that state. if that were the case, the DB wouldn't accept the uppercased variable names after i'd switched the db enum to lowercased - so we'd _have_ to use the `.value`.

but then i think there were other problems when trying to keep the variable names uppercased that motivated me to just switch them to lower case. and i kept these changes because they'd been in place and they seemed to make sense anyways (i.e., didn't break things or make the code less readable).

that being said, this doesn't really align with the functionality @seanpreston describes with `sqlalchemy` taking the `.value`. so the explanation may be wrong.

in summary, i'm not sure what motivated the change. but at this point, since we agree it doesn't break anything and isn't any less readable, we'll just keep it. sorry for the distraction on this one!